lib/ReadsUtils/ReadsUtilsImpl.py

```diff
@@ -125,7 +126,7 @@ def _proc_upload_reads_params(self, params):
             kbtype = 'KBaseFile.PairedEndLibrary'
             single_end = False
         if revid:
-            interleaved = 0
+            interleaved = 1
```

Comment:
My main issue was the separation of the interleaved logic into two parts lines apart when there's no need to do that and it makes understanding the interleaved logic harder. AFAIC you can use any if/else structure you want, just do the processing for the variable in one place when you can.